pylintrc

```diff
@@ -0,0 +1,328 @@
+[MASTER]
```

Comment:
Removing pylintrc file; I think this was mistakenly added anyway - we seem to have settled on .pylintrc being what we are using amongst travis-enabled projects